### DISCUSSION TITLE
Delete all snips at one time?

### DISCUSSION THREAD

**Discussion by stasimate** [robherley/snips.sh#79] [Q&A]
Is there any way to delete all snips at one time?

<img width="1680" alt="image" src="https://github.com/robherley/snips.sh/assets/74570672/dacd8b7b-d117-4bc0-97ef-ee6eb4ce3a56">

**Comment by robherley** [comment]
👋 I made an issue for this:
- https://github.com/robherley/snips.sh/issues/107

I'll see if I can find time to implement it, or maybe someone else can pick it up 👍

**Reply by CyberCitizen01** [reply]
Hey @robherley 👋🏻 
I believe this feature would be a nice addition. Moreover, looks like someone has stepped up to take this on. (https://github.com/robherley/snips.sh/issues/107#issuecomment-2137900762)
